src/components/Message/MessageSimple/MessageContent.js

```diff
@@ -554,18 +546,18 @@ MessageContent.propTypes = {
    *      onLongPress={(message, e) => {
    *        props.openReactionPicker();
    *        // Or if you want to open action sheet
-   *        // showActionSheet();
+   *        // props.showActionSheet();
```

Comment:
showActionSheet or onShowActionSheet? @DanC5 

Can we keep it as `showActionSheet` actually?

Reply:
just updated and synced up the naming